theLudovyc/Game2D.gd

```diff
@@ -9,11 +9,11 @@ class_name Game2D
 
 @onready var node_entities := %Entities
 
-@onready var the_storage := $TheStorage
+@onready var the_storage: TheStorage = $TheStorage
```

Comment:
Type is inferred here. Please see https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#variables

Reply:
Using static typing to help with the autocompletion in the IDE.

Reply:
Okay maybe it can help on the @onready var but please try to add just the necessary. GdScript is a scripting language, dynamically and weakly typed. I know it can be hard for a Java programmer, but try your best please :blush: